framework/PageToolbar/PageToolbarFilter.tsx

```diff
@@ -295,6 +296,18 @@ function ToolbarFilterComponent(props: {
           queryErrorText={filter.queryErrorText}
           queryPlaceholder={filter.queryPlaceholder}
           isRequired={filter.isRequired}
+          footer={
```

Comment:
This looks like it would add a browse button even if the filter did not have an `openBrowse` function hooked up.

Reply:
Aahh, yes, my bad, thanks for catching that.